### PR TITLE
Merge release-0.17 back

### PR DESCRIPTION
**Pull request by metalmatze**
Merging the `release-0.17` back into `master`.

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.16.0-dev
\ No newline at end of file
+0.17.0-dev
```
Comment (bwplotka) [inline VERSION:1]: I think you might be right @metalmatze 

Maybe 0.18.0-dev makes more sense? (:

**Review by bwplotka — APPROVED** [review]
:muscle: 

Thanks!  I think I like your idea more with master VERSION (: 

Hopefully we did not add anything new to change log in mean time.

File: VERSION (hunk 1 above)
Comment (metalmatze) [reply VERSION:1]: Updating!

**Comment by metalmatze** [comment]
Goutham changed something in the changelog and I just saw it's broken. Fixing :joy:

**Review by bwplotka — APPROVED** [review]
Thanks!

LGTM :hugs:

**Pull request merged**